src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -217,13 +258,13 @@ private HttpRequestMessage CreateRequestMessage(Uri endpoint, IDictionary<string
 
                 Stopwatch sw = Stopwatch.StartNew();
 
-                HttpClient client = GetHttpClient();
+                HttpClient client = GetHttpClient(bindingCertificate);
```

Comment:
Pls add more logging to the logging statement, i.e. if `bindignCertificate` is null or not.